enzyme/Enzyme/Utils.cpp

```diff
@@ -632,14 +637,29 @@ Function *getOrInsertDifferentialFloatMemcpy(Module &M, Type *elementType,
   return F;
 }
 
+Function *getOrInsertMemcpyStrided(Module &M, PointerType *T, Type *IT,
+                                   BlasInfo blas) {
+  std::string name =
+      (blas.prefix + blas.floatType + "copy" + blas.suffix).str();
+
+  FunctionType *FT =
+      FunctionType::get(Type::getVoidTy(M.getContext()), {T, T, IT, IT}, false);
+
+  Function *F = cast<Function>(M.getOrInsertFunction(name, FT).getCallee());
+  return F;
+}
+
 Function *getOrInsertMemcpyStridedBlas(Module &M, PointerType *T, Type *IT,
                                        BlasInfo blas, bool julia_decl) {
```

Comment:
Can we get rid of this function now?

Reply:
Yes, also of getOrInsertMemcpyStrided with Blas Parameter